src/main/java/karuna/karuna_backend/Controllers/ContentController.java

```diff
@@ -0,0 +1,27 @@
+package karuna.karuna_backend.Controllers;
+
+import karuna.karuna_backend.Models.Content;
+import karuna.karuna_backend.Services.ContentService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.HashMap;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/content")
+public class ContentController {
+
+    private final ContentService contentService;
+
+    @GetMapping("/{page}")
+    public ResponseEntity<HashMap<String, String>> getContentByPage(@PathVariable String page) {
```

Comment:
ResponseEntity u should use if u manipulate response, if u just returning ok and body it is not needed.
Here u have discuscion when to use: https://stackoverflow.com/questions/26549379/when-use-responseentityt-and-restcontroller-for-spring-restful-applications
For me, return type should be hashmap, @RestController change it to Response Body

Reply:
And to don't broke solid rules in feautre, for sure O will be brioken, wrapi in dto class

Reply:
What do you think to return List<ContentDTO> instead of HashMap<String, String>?
I talked to Bartek and maybe it would be better to return List<ContentDTO> if, for example, we wanted to add an ENG value in the future.

Also sorry but I don't understand why should I change RestController to ResponsyBody?  I'll leave it as is for now

Reply:
I believe he meant, that you should chang iResponseEntity in @RestController and return plain object instead. Therefore instead of public ReponseEneity<HashMap<String,String>> it would be plainly -> public ContentDTO getContentByPage(@PathVariable String page) {}

Reply:
And speaking of this one

> What do you think to return List instead of HashMap<String, String>? I talked to Bartek and maybe it would be better to return List if, for example, we wanted to add an ENG value in the future.
> 
> Also sorry but I don't understand why should I change RestController to ResponsyBody? I'll leave it as is for now

I've been talking to Karol (misterrevo) and we both agreed upon the fact that returning Dto List might be better idea